lib/log.dart

```diff
@@ -1,9 +1,14 @@
+import 'package:flutter/foundation.dart';
 
 /// Whether [debugLog] should do anything.
 ///
 /// This has an effect only in a debug build.
 bool debugLogEnabled = false;
 
+/// Used for log inspection in tests.
+@visibleForTesting
+List<String>? logHistory;
```

Comment:
```suggestion
List<String>? debugLogHistory;
```

This will live in our global namespace (e.g. it'll appear in autocomplete from all over the app) so the "debug" prefix is important context.